src/assets/images/aboutUs/aboutUsImages.js

```diff
@@ -0,0 +1,7 @@
+import aboutImg1 from "./aboutImg-1.png";
```

Comment:
I think we don't need this file. We can make image imports directly to component file, and it will have the same count of lines for `import` statements

Reply:
Done